kamon-annotation-api/src/main/java/kamon/annotation/api/CustomizeInnerSpan.java

```diff
@@ -24,24 +24,25 @@
 
 
 /**
- * A marker annotation to allows users to customize and add additional information to Spans created by instrumentation.
- * <p>
- * <p>
- * Given a method like this:
- * <pre><code>
- *     {@literal @}SpanCustomizer(operationName = "coolName")
- *     public String callDatabase(String query) {
- *         return Db.executeQuery(query);
- *     }
- * </code></pre>
+ * Indicates that any Spans created within the scope of the annotated method should have the customizations provided by
+ * this annotation. Currently the only supported customization is to change the operation name. For example, given a
+ * method like this:
+ *
+ * <pre>{@code
+ * @CustomizeSpan(operationName = "coolName")
```

Comment:
shoul be `CustomizeInnerSpan`?

Reply:
Ouch! Fixed it. Also updated the README and reference config :smile: